src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -195,7 +195,7 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getJavaToolsJar() {
+  public Path getJavaToolsJar() {
```

Comment:
1. Maybe the naming is wrong, but it's just a reference to the location of appcfg.

2. I think the actual expectation is that no one uses these. They really should be internal. Do you guys want to use this for something more than just testing?